### PR TITLE
Custom Customer creation for on prem installations

### PR DESCRIPTION
**Pull request by cihangir**

### CODE REVIEW

File: go/src/socialapi/workers/payment/api/customer_test.go [hunk 1]
```diff
@@ -197,3 +197,37 @@ func TestInfoPlan(t *testing.T) {
 		})
 	})
 }
+
+func TestCreateCustomCustomer(t *testing.T) {
+	Convey("Given a user", t, func() {
+		withTestServer(t, func(endpoint string) {
+			withTestCreditCardToken(func(token string) {
+				Convey("We should be able to create a custom user", func() {
+					cp := &stripe.CustomerParams{
```
Comment (cihangir) [inline go/src/socialapi/workers/payment/api/customer_test.go]: @usirin New endpoint accepts the arguments in this form
The URL is `koding.com/api/social/payment/custom-customer/create`

File: go/src/socialapi/workers/payment/api/customer.go [hunk 2]
```diff
@@ -68,6 +70,47 @@ func CreateCustomer(u *url.URL, h http.Header, req *stripe.CustomerParams, conte
 	)
 }
 
+// CreateCustomCustomer creates a custom customer for a group
+func CreateCustomCustomer(u *url.URL, h http.Header, initial *stripe.CustomerParams) (int, http.Header, interface{}, error) {
+	if initial.Email == "" {
+		return response.NewBadRequest(errors.New("email is not set"))
```
Comment (usirin) [inline go/src/socialapi/workers/payment/api/customer.go]: [nitpick] other error messages doesn't have `is` in the sentence.

File: go/src/socialapi/workers/payment/api/customer_test.go (hunk 1 above)
Comment (usirin) [reply go/src/socialapi/workers/payment/api/customer_test.go]: 💯

**Review by rjeczalik — APPROVED** [review]

**Comment by cihangir** [comment]
ref: https://github.com/koding/koding/pull/11461

**Comment by cihangir** [comment]
@usirin is this good to go?

**Comment by usirin** [comment]
Just a reminder, we need the nginx changes first for this to be merged.

**Review by usirin — APPROVED** [review]
:100:

**Pull request merged**